packages/service-provider-core/src/connect-info.ts

```diff
@@ -57,9 +58,22 @@ export default function getConnectInfo(
     is_enterprise: getBuildInfo.isEnterprise(buildInfo),
     auth_type,
     is_data_federation,
+    is_stream: isStream(uri),
     dl_version,
     atlas_version: atlasVersion?.atlasVersion ?? null,
     is_genuine,
     non_genuine_server_name,
   };
 }
+
+function isStream(uri: string): boolean {
```

Comment:
NIT: if you have the time for one additional PR: it would be awesome to have this function in https://github.com/mongodb-js/mongodb-build-info where we have all the other similar types of detection, so we will have it also for the other tools.

Reply:
That did cross my mind. Happy to move this to `mongodb-build-info`. Will raise a PR there.

Reply:
Have raised PR https://github.com/mongodb-js/mongodb-build-info/pull/23.